css-brochure-form/styles.css

```diff
@@ -0,0 +1,72 @@
+body {
+  font-family: sans-serif, "Brandon Grotesque";
+  font-style: normal;
+  font-weight: normal;
+  src: url("BrandonGrotesque-Regular.woff2") format("woff2");
```

Comment:
These are only necessary for the @font-face rule, not using the font. You can just state the name of the fonts in the `fonts.css` file in the `font-family` property values to use it.